src/particles.zig

```diff
@@ -141,15 +141,14 @@ pub const ParticleSystem = struct {
 	var particlesLocal: [maxCapacity]ParticleLocal = undefined;
 	var properties: EmmiterProperties = undefined;
 	var seed: u64 = undefined;
+	var previousPlayerPos: Vec3f = undefined;
```

Comment:
Should probably be Vec3d, otherwise you get precision issues when calculating the distance